sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -1095,6 +1097,23 @@ export class FeedResponse<TResource> {
     readonly resources: TResource[];
 }
 
+// @public
+export interface FullTextIndex {
```

Comment:
FullTextSearchIndex, FullTextSearchPath, FullTextSearchPolicy might be clearer, just `fullText` doesn't tell me much